crates/sdk-core/src/worker/workflow/machines/workflow_machines.rs

```diff
@@ -811,7 +800,12 @@ impl WorkflowMachines {
                             let resps = lam.try_resolve_with_dat((*la_dat).into())?;
                             self.process_machine_responses(mk, resps)?;
                         } else {
-                            self.local_activity_data.insert_peeked_marker(*la_dat);
+                            // Since the LA machine exists, we have encountered the LA WF command.
```

Comment:
Is this case even possible? If it is, it feels wrong, and possibly dangerous, to simply drop the marker.

Unless we can prove this case exists, or otherwise prove that dropping the marker would be safe, I think we should just remove the call to `will_accept_resolve_marker` above, and let `try_resolve_with_dat` propagate an error should we ever reached that from an unexpected state.